NBitcoin/RPC/RPCClient.cs

```diff
@@ -1181,6 +1181,11 @@ public async Task<BlockchainInfo> GetBlockchainInfoAsync()
 			return blockchainInfo;
 		}
 
+		public BlockchainInfo GetBlockchainInfo()
+		{
+			return GetBlockchainInfoAsync().Result;

```

Comment:
`.GetAwaiter().GetResult()` so the exception does not get wrapped inside `AggregateException`